xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -230,7 +230,10 @@
       "language": { "type": "string", "required": true },
       "codec": { "type": "string", "required": true },
       "bitrate": { "type": "integer", "required": true },
-      "channels": { "type": "integer", "required": true }
+      "channels": { "type": "integer", "required": true },
+      "is_default": { "type": "boolean", "default": false, "required": true },
```

Comment:
Please remove the "default" field from these new properties, then we'll be good to go.

Reply:
Done thanks!